docs/gallery/gallery.yml

```
@@ -344,7 +344,16 @@
       href: https://nhsrcommunity.com
       code: https://github.com/nhs-r-community/nhs-r-community
       thumbnail: websites/nhs-r-website.png
-
+    - title: Mickaël Canouil Universe
+      subtitle: A data science blog and portfolio by Mickaël CANOUIL
+      href: https://mickael.canouil.fr
+      code: https://github.com/mcanouil/mickael.canouil.fr
+      thumbnail: websites/website-mickael-canouil.fr.png
```

```suggestion
      thumbnail: websites/website-mickael.canouil.fr.png
```

This is the image path committed in the PR. so either modifying here, or renaming the image.